### PR TITLE
65 cart contex fix

### PR DESCRIPTION
**Pull request by liinneea-a**

### CODE REVIEW

File: src/components/AddToCartSnackbar.tsx [hunk 1]
```diff
@@ -4,52 +4,60 @@ import Snackbar, { SnackbarOrigin } from '@mui/material/Snackbar';
...
+		getTotalSum,
 		addQuantity,
 		reduceQuantity,
-		// totalQuantity,
+		getTotalQuantity,
 		createOrderId,
 		calculateVatPrice,
 	} = useCart();
 
-	const handleClick = (newState: SnackbarOrigin) => () => {
```
Comment (millifrill) [inline src/components/AddToCartSnackbar.tsx:34]: You can't remove this

File: src/components/AddToCartSnackbar.tsx [hunk 2]
```diff
@@ -4,52 +4,60 @@ import Snackbar, { SnackbarOrigin } from '@mui/material/Snackbar';
...
-		setState({ open: true, ...newState });
+	const handleOnClickAdd = () => {
+		
+		// setState({ open: true, vertical: 'top', horizontal: 'right' });
+		addProductToCart(props.product);
+		console.log('snackbarAdd')
 	};
 
-	const handleClose = () => {
-		setState({ ...state, open: false });
```
Comment (millifrill) [inline src/components/AddToCartSnackbar.tsx:39]: You can't remove this

File: src/components/AddToCartSnackbar.tsx [hunk 3]
```diff
@@ -4,52 +4,60 @@ import Snackbar, { SnackbarOrigin } from '@mui/material/Snackbar';
...
+	const handleOnClickClose = () => {
+		setPopUpState({ ...popUpState, open: false });
 	};
 
 	const buttons = (
 		<React.Fragment>
-			<Button
-				onClick={handleClick({
```
Comment (millifrill) [inline src/components/AddToCartSnackbar.tsx:45]: You can't remove this

File: src/components/AddToCartSnackbar.tsx [hunk 4]
```diff
@@ -60,7 +68,7 @@ export default function AddToCartSnackbar() {
 				anchorOrigin={{ vertical, horizontal }}
 				open={open}
 				autoHideDuration={2000}
-				onClose={handleClose}
```
Comment (millifrill) [inline src/components/AddToCartSnackbar.tsx:63]: You can't remove this

**Review by millifrill — CHANGES_REQUESTED** [review]

**Comment by millifrill** [comment]
You have remove functionality on snackbar

File: src/components/AddToCartSnackbar.tsx (hunk 3 above)
Comment (liinneea-a) [reply src/components/AddToCartSnackbar.tsx:45]: The button is making it not work

File: src/components/AddToCartSnackbar.tsx (hunk 2 above)
Comment (liinneea-a) [reply src/components/AddToCartSnackbar.tsx:39]: not removed. only changed names

File: src/components/AddToCartSnackbar.tsx (hunk 4 above)
Comment (liinneea-a) [reply src/components/AddToCartSnackbar.tsx:63]: not removed. changed the name

**Review by millifrill — APPROVED** [review]

**Review by empafrontend — APPROVED** [review]

**Pull request merged**